### PR TITLE
Add testcases of type i8, i16, u8, u16, bf16, and f64 for TopK

### PR DESCRIPTION
**Pull request by steve-y**
### Details:
 - *Add template plugin testcases of i8, i16, u8, u16, bf16, and f64 to op_reference/topk.cpp*
 - *Remove skip regex from skip_tests_config.cpp*
 - *Add types support to op/topk.cpp*

### Tickets:
 - *70525*

### CODE REVIEW

**Comment by steve-y** [comment]
- 1dMaxMin, Int64, and SingleOutput testcases include types of i8, i16, u8, u16, bf16, and f64.
- Resnet50, MaxMinSort, Backend testcases include types of i16, u16, bf16, and f64 because i8 and u8 generate overflow in datasets.

File: src/core/src/op/topk.cpp [hunk 1]
```diff
@@ -91,12 +91,18 @@ bool evaluate_topk(const HostTensorPtr& arg,
                    const element::Type index_et) {
     bool rc = true;
     switch (arg->get_element_type()) {
+        NGRAPH_TYPE_CASE(evaluate_topk, i8, arg, out_indices, out_values, out_shape, axis, k, max, sort, index_et);
```
Comment (ilyachur) [inline src/core/src/op/topk.cpp:94]: Why did you add new data types to topK? we should extend evaluate only in case of customer models, because it increases the binary size.

Comment (steve-y) [reply src/core/src/op/topk.cpp:94]: I got it as a bug, and as I got it as ordinary, I'll close this PR.
